src/index.tsx

```diff
@@ -134,70 +134,70 @@ function exportAsPNG({
   // deselect & rerender
 
   clearSelection();
-  drawScene();
+  ReactDOM.render(<App />, rootElement, () => {
```

Comment:
drawScene was synchronous, but rendering is not, so need to move everything in a callback.